adb_graphics/figures/skewt.py

```diff
@@ -130,6 +133,26 @@ def _add_hydrometeors(self, hydro_subplot): # pylint: disable=too-many-locals
                                marker=settings.get('marker'),
                                markersize=6,
                               )
+            if mixr in ['clwmr', 'rwmr']:
+                hydro_subplot.plot(profile[temp.magnitude < 32.0], pres[temp.magnitude < 32.0],
+                                   settings.get('color'),
+                                   fillstyle='full',
+                                   linewidth=0.5,
+                                   marker=settings.get('marker'),
+                                   markersize=6,
+                                  )
+                layer = False
+                for i in range(len(profile)):
```

Comment:
Using enumerate here cleans things up a bit:
```
for i, vertical_lev in enumerate(profile):
    if ((vertical_lev > 0.0 and temp[i].magnitude < 32.0) and not layer):
    ...
```

Reply:
Thanks for showing the proper syntax, I hadn't gotten it right last night.  I've made the change and "un-disabled" the pylint check.